src/common/cockpittest.c

```diff
@@ -220,6 +220,9 @@ cockpit_test_init (int *argc,
   g_assert (g_snprintf (path, sizeof (path), "%s:%s", BUILDDIR, g_getenv ("PATH")) < sizeof (path));
   g_setenv ("PATH", path, TRUE);
 
+  /* For our process and for children */
```

Comment:
This doesn't apply to children, would need an environment variable. Perhaps @martinpitt approach is better. Although we should be aware that it doesn't work when launching the tests as individual processes to figure out an issue, or run them with gdb ... so worth considering using both approaches.